.goreleaser.yml

```diff
@@ -0,0 +1,56 @@
+before:
+  hooks:
+    - go mod download
+
+builds:
+  - main: ./cmd/slo_exporter.go
+    binary: slo_exporter
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - 386
+
+archives:
+  - replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
```

Comment:
(In light of our today's discussion) I'd remove whole `archives` section.

Reply:
Yes, removed this section completely